frontend/src/registration/components/DomainInputPage/displayMessages.ts

```diff
@@ -1,12 +1,4 @@
 const messages = {
-  typeDomainNameBelow: {
-    defaultMessage: 'Type your domain name below',
-    description: 'Message above domain input text box.'
-  },
-  checkAvailability: {
-    defaultMessage: 'Check Availability',
-    description: 'Label for button that check availability of Domain'
-  },
```

Comment:
Won't we use it later for own domains?

Reply:
This was moved to `ui/components`, the file is still there. :)

Reply:
@giovannicimolin, oh, sorry, I missed this :)